### PR TITLE
replace obsolete MONDO term with replacement term

### PR DESCRIPTION
**Pull request by sierra-moxon**
Replaced term using the suggested replacement from Mondo

### CODE REVIEW

File: test_cases/TestCase_32.json
Comment (colleenXu) [inline test_cases/TestCase_32.json:1]: @sierra-moxon something seems odd with this file....maybe a merging problem? It seemed to me like assets 324-326 were in the TestCase_47.json file...

**Comment by colleenXu** [comment]
And that file's edits end up affecting the test suite files (generated automatically?). 

It looks a little weird to me, like a bunch of "diabetes" tests from TestCase_32.json are being changed right now to interstitial lung disease....when I'd instead expect the idiopathic pulmonary fibrosis lines to be changed.

**Comment by colleenXu** [comment]
Note: this is to address https://github.com/NCATSTranslator/Tests/issues/35

File: test_cases/TestCase_32.json
Comment (sierra-moxon) [reply test_cases/TestCase_32.json:1]: that is weird!  I wonder why I didn't get a conflict here.  Fixing.

**Comment by sierra-moxon** [comment]
After some investigation, we have a Case/Asset mismatch in the main branch that is causing this strangeness with my replacement of "MONDO:0008345". 

TestCase_32 in sprint_4_tests.json is "what affects CHEBI:167574" with "semaglutide" in the answers 
https://github.com/NCATSTranslator/Tests/blob/414b240ca84cb3d64e09650b55d1bf133ad299c0/test_suites/sprint_4_tests.json#L6266

TestCase_32 in semantic_smoke_test_suite_PROD.json is "what treats MONDO:0008345", "Pirfenidone treats Idiopathic pulmonary fibrosis" https://github.com/NCATSTranslator/Tests/blob/414b240ca84cb3d64e09650b55d1bf133ad299c0/test_suites/semantic_smoke_test_suite_TEST.json#L16066

TestCase_32 itself in test_cases/TestCase_32.json is "what treats MONDO:0005015", "treats Diabetes Mellitus" https://github.com/NCATSTranslator/Tests/blob/main/test_cases/TestCase_32.json

File: test_cases/TestCase_32.json
Comment (sierra-moxon) [reply test_cases/TestCase_32.json:1]: https://github.com/NCATSTranslator/Tests/issues/90

**Review by colleenXu — APPROVED** [review]

**Comment by colleenXu** [comment]
I'm not sure if the asset backup files need to be adjusted (is it helpful to remember that previously the ID was different?). 

But I approved this anyways. I don't have write access, so it looks like my approval isn't enough to allow a merge.

**Review by maximusunc — APPROVED** [review]

**Pull request merged**
